resolvers/event.js

```diff
@@ -5,8 +5,9 @@ const validators = require('../utils/validators');
 
 const eventResolvers = {
   Query: {
-    events: () =>
-      db.event.findAll({
+    events: () => {
+      validators.validateRole(['COMMANDER', 'SUPERVISOR', 'DISPATCH']);
```

Comment:
`role` is currently an array of strings, ideally we would want to work with enums here instead

if it doesn't make sense to make the change now, would need to be tackled with authentication by @JasonYG

Reply:
I'll discuss with Jason regarding this.